packages/compass-aggregations/src/components/stage-toolbar/stage-operator-select.tsx

```diff
@@ -156,7 +148,6 @@ export default connect(
         'serverVersion',
         'env',
         'isTimeSeries',
-        'isReadonly',
```

Comment:
`isReadonly` was not used here (and in the agg plugin in general), so I removed it